revolv/project/tests/test_integration.py

```diff
@@ -0,0 +1,185 @@
+import mock
+import json
+from django.test import TestCase
+from django_webtest import WebTest
+from revolv.lib.testing import TestUserMixin, UserTestingMixin
+from revolv.project.models import Project, ProjectUpdate
+
+
+class PostProjectUpdatesTest(TestUserMixin, UserTestingMixin, TestCase):
+
+    def helper_test_create_update_ambassador_administrator(self):
+        """
+        Tests that administrators and ambassadors can create project updates.
+        """
+        response = self.send_test_user_login_request()
+        self.assertUserAuthed(response)
```

Comment:
If you're doing asserts in this helper function, then its name should start with `assert`. For this function, looks like `assert_user_can_create_project_update` would be a good name